test/plugin/test_out_bigquery.rb

```diff
@@ -695,28 +652,27 @@ def test_format_with_insert_id
       insert_id_field uuid
       field_string uuid
     CONFIG
-    driver.instance.start
-    buf = driver.instance.format_stream("my.tag", [input])
-    driver.instance.shutdown
-
-    assert_equal expected, MessagePack.unpack(buf)
+    mock(driver.instance).insert("foo", [expected], nil)
+    driver.instance_start
```

Comment:
remove this (controllable by keyword arguments of `#run`)
https://github.com/fluent/fluentd/blob/master/lib/fluent/test/driver/base.rb#L70